CI/functional_tests/test_water_study.py

```diff
@@ -0,0 +1,159 @@
+    water.get_file(temporary_path)
+    file_paths = [(temporary_path / f).as_posix() for f in water.file_raw]
+    return file_paths
+
+
+@pytest.fixture()
+def mdsuite_project(traj_files, tmp_path) -> mdsuite.Project:
+    """
+    Create the MDSuite project and add data to be used for the rest of the tests.
+
+    Parameters
+    ----------
+    traj_files : list
+            Files include:
+                * Water Simulation
+    tmp_path : Path
+            Temporary path that may be changed into.
+
+    Returns
+    -------
+    project: mdsuite.Project
+            An MDSuite project to be tested.
+    """
+    gmx_units = Units(
+        time=1e-15,
+        length=1e-10,
+        energy=1.6022e-19,
+        NkTV2p=1.6021765e6,
+        boltzmann=8.617343e-5,
+        temperature=1,
+        pressure=100000,
+    )
+    project = mds.Project(storage_path=tmp_path.as_posix())
+
+    file_reader = mdsuite.file_io.chemfiles_read.ChemfilesRead(
+        traj_file_path=traj_files[2], topol_file_path=traj_files[0]
+    )
+
+    project.add_experiment(
+        name="water_sim",
+        timestep=0.002,
+        temperature=300.0,
+        units=gmx_units,
+        simulation_data=file_reader,
+    )
+    project.run.CoordinateUnwrapper()
+    return project
+
+
+def test_water_analysis(mdsuite_project):
+    """
+    Run a functional test by performing a study on an MD simulation of water
+
+    Notes
+    -----
+    The diffusion, angle, and eventually coordination data tested here are comparable
+    with values taken from experiment and published studies.
+    """
+
+    water = mdsuite_project.experiments["water_sim"]
+
+    water_molecule = mds.Molecule(
+        name="water", smiles="[H]O[H]", amount=14, cutoff=1.7, mol_pbc=True
+    )
+
+    water.run.MolecularMap(molecules=[water_molecule])
+    atomistic_adf = mdsuite_project.run.AngularDistributionFunction(plot=False)
+    mdsuite_project.run.RadialDistributionFunction(plot=False)
+    molecule_adf = mdsuite_project.run.AngularDistributionFunction(
+        plot=False, molecules=True, norm_power=8
+    )
+    mdsuite_project.run.RadialDistributionFunction(plot=False, molecules=True)
+    atomistic_diffusion = mdsuite_project.run.EinsteinDiffusionCoefficients(
+        plot=False, data_range=500
+    )
+    molecular_diffusion = mdsuite_project.run.EinsteinDiffusionCoefficients(
+        plot=False, molecules=True, data_range=500
+    )
+
+    water_group = water.molecules["water"]["groups"]["0"]
+    atom_group_adf = mdsuite_project.run.AngularDistributionFunction(
+        atom_selection=water_group, number_of_configurations=100, plot=False
+    )
+    atom_group_diffusion = mdsuite_project.run.EinsteinDiffusionCoefficients(
+        atom_selection=water_group, data_range=2500, plot=False
+    )
+
+    # Test ADF output
+    assert atomistic_adf["water_sim"].data_dict["O_H_H"]["max_peak"] == pytest.approx(
```

Comment:
@PythonFZ would have to answer that, but in principle no it can't arbitrarily be changed here.

Reply:
Maybe - we should go over to not use strings but frozensets idealy for this. Any other suggestions on how to handle that? We could also instead of a dict add a method or allow for all permutations?

Reply:
In the case of an ADF there is order here actually. The O is the central atom and there exists another called "H_O_H" where the Hydrogen is central and bonded between one oxygen and another hydrogen. So in the case of the ADF, this is no arbitrary. For RDFs however I thought it depended how it was loaded from the database

Reply:
my non-chemist brain is overloaded by the thought of an H-atom binding to two other atoms. 
In the cases where it is arbitrary, could we sort alphabetically to make it somewhat deterministic?

Reply:
Moved to #486